xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -746,10 +794,20 @@
         <Parameter Name="item" Type="System.Text.RegularExpressions.Capture" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="index">The zero-based index at which <paramref name="item" /> should be inserted.</param>
+        <param name="item">The object to insert into the collection.</param>
+        <summary>Inserts an item to the collection at the specified index.</summary>
+        <exception cref="T:System.NotSupportedException">The collection is read-only.</exception>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If `index` equals the number of items in the <xref:System.Collections.Generic.IList%601>, then `item` is appended to the list.   
```

Comment:
I think this remark got ported from the interface. Maybe we should adapt it for this API. Something like:
```suggestion
If `index` equals the number of items in this <xref:System.Collections.Generic.IList%601> instance, then `item` is appended to the list.   
```
Feel free to reword it.

Reply:
It feels weird saying anything here other than "This throws a `NotSupportedException` all the time. The collection is read-only."  Saying anything else contradicts what this method does.

Reply:
Oh! If that's the case, the remarks should be empty, and the summary should say it throws all the time.

@pgovind if you find more cases like this, make sure to edit them the same way.

Reply:
Fixed.